code/modules/mob/living/simple_animal/hostile/alien.dm

```diff
@@ -30,7 +30,7 @@
 	see_in_dark = 8
 	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
 	unique_name = 1
-	gold_core_spawnable = NO_SPAWN
+	gold_core_spawnable = HOSTILE_SPAWN
```

Comment:
Terrible idea, but you know what you are getting into when you use gold cores so I approve.

Reply:
:joker: